app/templates/apps/details.html

```diff
@@ -20,147 +15,230 @@ <h2 class="heading-medium">Repo</h2>
   {% endif %}
 </p>
 
-<h2 class="heading-medium">App admins</h2>
-{% if app.userapps.length %}
-  <table class="app-admins form-group">
-    <thead>
-      <tr>
-        <th>Name</th>
-        <th><span class="visuallyhidden">Actions</span></th>
-      </tr>
-    </thead>
-    <tbody>
-      {% for userapp in app.userapps %}
+<h2 class="heading-medium">Description</h2>
+<p>
+  {{ app.description if app.description else "None" }}
+</p>
+
+
+<div class="form-group">
+  <h2 class="heading-medium">App customers <em class="js-modal-trigger" data-content="app-customers"><span>What's this?</span></em></h2>
+  <div class="js-hidden js-modal-content" id="content-app-customers">
+    <h3 class="heading-small">App customers</h3>
+    <p>App customers are end users of a webapp. They do not have to be Control Panel users, any email address can be added.</p>
+  </div>
+  {% if customers.length %}
```

Comment:
Does this not work as `if customers`?

Reply:
Oh, even in nunjucks?

Reply:
Yep, they diverge from Jinja2 on this - ok, never mind.

Reply:
👍